styles/modules.css

```diff
@@ -0,0 +1,37 @@
+/*#main-nav li:hover{
```

Comment:
In this file I would suggest focusing on smaller components: your nav bar, handlebars styling, etc. I would include both all views and @media only view.